Dockerfile

```diff
@@ -47,5 +47,6 @@ unzip ./chromedriver_linux64.zip
 RUN poetry install --no-root
 COPY . .
 RUN chmod +x ./tests.sh
+ENV SECRET_KEY=TEST_KEY

```

Comment:
It would be more consistent to pass this through as any other environment variables, e.g. in your `.env.test`/`.env`, or through Travis directly